src/lib/create-config.ts

```diff
@@ -198,7 +202,7 @@ const loadTokensFromString = (tokenString: String, tokenType: ApiTokenType) => {
             type: tokenType,
             username: 'admin',
         };
-        validateApiToken(token);
+        validateApiToken(mapLegacyToken(token));
```

Comment:
Ok, so, we won't have support for adding multiproject tokens from environment variables for this release.

Reply:
That is correct! We need to come up with a separate format for the env vars, currently they're determined from the secret signature in this code path. The secret doesn't contain all the info needed for a multiproject. I think something like this would work:
 
["default", 'new-project']:7fae6baeff 

We can read that string, calculate the projects and then strip them